.github/workflows/ci.yml

```diff
@@ -78,7 +78,7 @@ jobs:
           echo Run ctest -V -L DFLY
           #GLOG_logtostderr=1 GLOG_vmodule=transaction=1,engine_shard_set=1
           GLOG_logtostderr=1 GLOG_vmodule=rdb_load=1,rdb_save=1,snapshot=1 ctest -V -L DFLY
-          ./dragonfly_test  --mem_defrag_threshold=0.05 # trying to catch issue with defrag
+          ./dragonfly_test  --gtest_repeat=100
```

Comment:
@romange --gtest_repeat=100 it seems 100 repetitions timesout the ci-tests (takes over 30 mins), is it needed?